terraform/main.tf

```diff
@@ -0,0 +1,117 @@
+provider "aws" {
+  region = "us-east-1"
+  profile = "serverless"
+}
+
+data "aws_vpc" "default" {
+  vpc_id = "vpc-0d60ef77"
+}
+
+data "aws_subnet_ids" "all" {
+  vpc_id = data.aws_vpc.default.id
+}
+
+resource "aws_elasticache_cluster" "network-data" {
+  cluster_id           = "network-data"
```

Comment:
Won't this name confuse us a bit when seeing it in AWS Console?
I feel we should put something more specific like "hathor-explorer-service-network-data"

What do you think?

Reply:
Agree! Let's talk about it soon